organizations/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
 edx-organizations app initialization module
 """
-__version__ = '0.4.10'  # pragma: no cover
+__version__ = '0.4.11'  # pragma: no cover
```

Comment:
I don't think we can consider this as a bug fix. It would rather be a patch so I think this should change to `0.5.0`

Reply:
Sure! Done.